libraries/core-react/src/components/Select/SingleSelect/SingleSelect.tsx

```diff
@@ -40,6 +40,8 @@ export type SingleSelectProps = {
   /** Callback for the selected item change
    * changes.selectedItem gives the selected item
    */
+  /** Allow for filtering of options to be turned off */
```

Comment:
Suggest updating the description to include that its filtering of options when an option is already selected. Just to make it clear thats its not the filtering all together 🙈